src/Service/BuildService.php

```diff
@@ -280,8 +280,9 @@ public function createDuplicateBuild(Build $originalBuild, $source)
     public function deleteOldByProject($projectId)
     {
         $keepBuilds = (int)Config::getInstance()->get('php-censor.build.keep_builds', 100);
-        $builds     = $this->buildStore->getOldByProject((int)$projectId, $keepBuilds);
-        
+        $keepBatch  = (int)Config::getInstance()->get('php-censor.build.keep_builds_batchsize', 100);
```

Comment:
If you want to have `batch` parameter in the command you should define command argument (For example: `-s|--batch_size`). Global configuration is the bad place for the parameter.

The same goes for `keep_builds` parameter (`deleteOldByProject` use global config because it was for GUI).